types/loglevel.pp

```diff
@@ -0,0 +1,2 @@
+# @summary As per http://httpd.apache.org/docs/current/mod/core.html#loglevel
```

Comment:
Would `@see` be useful for the URL?

Reply:
I've had a go at making the puppet-strings based docs better. Changes to the REFERENCE.md aren't in this commit as it's very difficult to exclude unrelated changes.  It looks like the REFERENCE.md will be updated as part of the release process.